src/pudl/metadata/fields.py

```diff
@@ -287,10 +287,12 @@
         "type": "boolean",
         "description": "Is the value observed (True) or imputed (False).",
     },
-    "data_source": {
+    "data_maturity": {
         "type": "string",
-        "description": "Source of EIA 860 data. Either Annual EIA 860 or the year-to-date updates from EIA 860M.",
-        "constraints": {"enum": ["eia860", "eia860m"]},
+        "description": "Level of maturity of the data record. Some data sources report less-than-final data. PUDL sometimes includes this data, but use at your own risk.",
+        "constraints": {
+            "enum": sorted(set(CODE_METADATA["data_maturities"]["df"]["code"]))
```

Comment:
If you've set it up to have a FK constraint between all the `data_maturity` columns and this coding table with a FK rule (which it seems you have) then this ENUM is duplicative.

Reply:
ah got it! okay i'll remove this